examples/typeorm-basic-usage/entities/rating.ts

```diff
@@ -7,6 +7,7 @@ import {
   PrimaryGeneratedColumn,
   RelationId,
 } from "typeorm";
+// eslint-disable-next-line import/no-cycle
```

Comment:
Do you want to leave the inline eslint comments in examples or it's just temporary solution?